common/src/sim_config.rs

```diff
@@ -28,8 +28,22 @@ pub struct SimConfigRaw {
     pub no_clip_movement_speed: Option<f32>,
     /// Character maximumum movement speed while on the ground in m/s
     pub max_ground_speed: Option<f32>,
+    /// Character artificial speed cap to avoid overloading the server in m/s
+    pub speed_cap: Option<f32>,
+    /// Maximum ground slope (0=horizontal, 1=45 degrees)
+    pub max_ground_slope: Option<f32>,
     /// Character acceleration while on the ground in m/s^2
     pub ground_acceleration: Option<f32>,
+    /// Character acceleration while in the air in m/s^2
+    pub air_acceleration: Option<f32>,
+    /// Acceleration of gravity in m/s^2
+    pub gravity_acceleration: Option<f32>,
+    /// Linear air resistance in (1/e)-lives per second
```

Comment:
lives?

Reply:
It took me a while to remember what I meant by this, so I probably want to reword it. It is like "half-lives", but "1/e"-lives instead.

I'll reword this, as it's really just the factor applied to the velocity to get the acceleration caused by friction.